test/performance/compiler/bradc/cg-sparse-timecomp.graph

```diff
@@ -1,3 +1,3 @@
-perfkeys: parse :, checkParsed :, cleanup :, scopeResolve :, flattenClasses :, normalize :, checkNormalized :, buildDefaultFunctions :, resolve :, checkResolved :, flattenFunctions :, cullOverReferences :, parallel :, lowerIterators :, inlineFunctions :, scalarReplace :, copyPropagation :, deadCodeElimination :, prune :, insertLineNumbers :, codegen :, refPropagation :
+perfkeys: total time (front end) :, total time (middle end) :, total time (back end) :
```

Comment:
These are effectively historical tests at this point, so let's leave them alone. https://github.com/Cray/chapel-private/issues/3600#issuecomment-1218308729 has some more context, but the basic gist is that these 3 tests were our original compiler performance tests, but they were always out of date in terms of what passes were being tracked and were no longer good exemplar tests anymore, so we started tracking the average compilation time and added the (now annoying) infrastructure to fail if a new pass was added as a sentinel that something needed to change.

What I think you want to modify is the infra around that average compilation time reporting. So that's things like `test/performance/compiler/*{graph,perfkeys}` and probably a few other things in that {sub}directory. You should be able to test with `start_test --compperformance test/release/examples/ --no-recurse`.